src/Workspaces/Core/Portable/EmbeddedLanguages/VirtualChars/VirtualChar.cs

```diff
@@ -23,33 +25,49 @@ namespace Microsoft.CodeAnalysis.EmbeddedLanguages.VirtualChars
     /// </summary>
     internal readonly struct VirtualChar : IEquatable<VirtualChar>
     {
-        public readonly char Char;
+        public readonly Rune Rune;
         public readonly TextSpan Span;
 
-        public VirtualChar(char @char, TextSpan span)
+        public VirtualChar(char ch, TextSpan span)
```

Comment:
💭  Why are we keeping this constructor and not just using the other one in all cases?